assets/js/wcmp-frontend.js

```diff
@@ -178,7 +178,9 @@ jQuery( function( $ ) {
 		if (country != 'BE') {
 			hide_myparcelbe_delivery_options();
 		} else {
-			$( '#mypa-pickup-location-selector' ).show();
+            if(myParcelConfig.allowPickup){
```

Comment:
Vreemd, hier staan weer spaties, maar in het bestand tabs

Reply:
Done